### ISSUE TITLE
ikvmc fails with System.IO.FileNotFoundException: netstandard error

### ISSUE THREAD

**Issue by richardschoen** [ikvmnet/ikvm#322]
I'm trying to convert the jt400.jar file to jt400.dll using ikvmc. 

This works with the older classic .Net Framework versions, but when I tried to test with the x86 version of the new ikvmc I get the following error.

```netstandard.dll``` is in the directory path as I copied jt400.jar into the same directory as: 
```IKVM-8.5.0-prerelease.1-tools-netcoreapp3.1-win7-x86``` and changed to the directory from command prompt before running ikvmc

Command run:
```ikvmc jt400.jar```

Results:  
```--------------------------------- ```
IKVM.Tools.Importer (8.5.0-prerelease.1+1.Branch.release-8.5.Sha.98716065013b1e01aaea10f3981c2ff062670f29)
Copyright c 2023 Jeroen Frijters, Windward Studios, Jerome Haltom, Shad Storhaug

*** INTERNAL COMPILER ERROR ***

PLEASE FILE A BUG REPORT FOR IKVM.NET WHEN YOU SEE THIS MESSAGE

IKVM.Tools.Importer, Version=8.5.0.0, Culture=neutral, PublicKeyToken=13235d27fcbfff58
C:\Users\Richard\Downloads\ikvm8.5.0\
3.1.32 32-bit

```System.IO.FileNotFoundException: netstandard```
   at IKVM.Reflection.Universe.Load(String refname, Module requestingModule, Boolean throwOnError) in C:\work\ikvm\src\IKVM.Reflection\Universe.cs:line 547
   at IKVM.Tools.Importer.AssemblyResolver.Init(Universe universe, Boolean nostdlib, IList`1 references, IList`1 userLibPaths) in C:\work\ikvm\src\IKVM.Tools.Importer\AssemblyResolver.cs:line 96
   at IKVM.Tools.Importer.IkvmImporterInternal.Compile(String[] args) in C:\work\ikvm\src\IKVM.Tools.Importer\IkvmImporterInternal.cs:line 178
   at IKVM.Tools.Importer.IkvmImporterInternal.Execute(String[] args) in C:\work\ikvm\src\IKVM.Tools.Importer\IkvmImporterInternal.cs:line 112

```--------------------------------- ```  

All thoughts are welcome as to how I might resolve this. Thanks

**Comment by wasabii** [comment]
Have you attempted to use IkvmReference as mentioned in the README?

**Comment by richardschoen** [comment]
Most likely not. In the past I've not had to.  I will take a look.

**Comment by richardschoen** [comment]
Any chance there's a sample .Net Core project that uses IkvmReference that I could use as a template ?

Also are you saying that ikvmc doesn't work but IkvmReference should ?

Just want to make sure I'm understanding as in past versions prior to this reboot ikvmc just worked. 

Not trying to be sarcastic. Just want to make sure I don't spin my wheels on this. Thanks

**Comment by richardschoen** [comment]
If not I'm happy to create a little .Net Core reference project if I get things to work.

**Comment by wasabii** [comment]
We strongly recommend you work in this order: MavenReference, IkvmReference, and only if those do not work for you, ikvmc directly. I don't have complete documentation on the changes with ikvmc, but there's at least 10 issues at this point with the same general problem. People coming from the old IKVM, expecting to just run an exe, and have output.

I'll go find one.

The general gist though is you cannot any longer just specify the jar and expect it to work. You need to specify the -reference argument to point to every assembly required. Because in .NET Core, there is no longer a single source of truth for what you might be targeting. Are you targeting ref-only assemblies from .NET 6.0.104? Or .NET 5.1.200?

And you also need to specify the -runtime argument to point to the specific version of IKVM.Runtime you want.

Because you could be attempting to run ikvmc from .NET 5, targeting an IKVM.Runtime from version 8.4 compiled for .NET Core 3.1.

Basically, because the complexity of Core exploded, the complexity of every compiler has exploded as well. Same reason, for example, you can no longer just run csc.exe from .NET Core and expect it to build your assembly. You have to -reference three dozen different assemblies now.

**Comment by wasabii** [comment]
Is this what you're after?

https://mvnrepository.com/artifact/net.sf.jt400/jt400/11.2

```
<MavenReference Include="net.sf.jt400:jt400" Version="11.2" />
```

Should be all you need (if it works).

**Comment by richardschoen** [comment]
By the way it worked. Thanks again.
